packages/mds-stream/kafka/agency-stream-kafka.ts

```diff
@@ -1,7 +1,30 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
 import { VehicleEvent, Telemetry, Device } from '@mds-core/mds-types'
 import { getEnvVar } from '@mds-core/mds-utils'
-import { AgencyStream } from '../stream-interface'
-import { KafkaStreamProducer } from './stream-producer'
+import { KafkaStreamProducer } from '.'
+
+export interface AgencyStream {
```

Comment:
I'd like to keep this in a shared place -- I was going to reuse for the NATS client refactor that's currently in limbo.

Reply:
I can move it. Ideally, it ought to be in agency, but that's a bigger lift.

Reply:
I can move it. Ideally, it ought to be in agency, but that's a bigger lift.

Reply:
Moved to `agency-stream-interface.ts`